app/src/main/java/org/listenbrainz/android/model/Playable.kt

```diff
@@ -4,7 +4,8 @@ data class Playable(
     val type: PlayableType,
     val id: Long,
     val songs: List<Song>,
-    var currentSongIndex : Int
+    var currentSongIndex : Int,
+    var seekTo: Long = 0L
```

Comment:
What does this mean for a Playable  as the type playlist, for example?

Reply:
It is by default 0 even in the changePlayable function default value is given as zero, during a change in the queue list we need to use the function preparePlayer because without that users are able to see the change list in the queue but it will not play and in prepare player function is written in such a way that whenever it will be called, it start a song from the beginning, so by adding this when user change the playable list it also stores the current position of a song and use the preparePlayer function it starts a song from the where the user has listened.